### DISCUSSION TITLE
0.46.0 beta

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#257] [Announcements]
- options: tiles mode
- plugins: several new capabilities, [check history](https://github.com/rejetto/hfs/blob/main/dev-plugins.md#api-version-history)
- automatic backup of config
- [new "maskOnly" in VFS](https://github.com/rejetto/hfs/blob/main/config.md#virtual-file-system-vfs)
- new icons for image, video and audio files
- file menu: show #270 
- download progress #273 
- rename #251 
- new icons for image/video/audio
- file menu: uploader #271
- safer: prevent session cookie to be used by other ip
- safer: no csrf in log
- langs/vi
- langs/ru updated
- langs/fr updated
- plugin/download-counter: ignore option
- plugin/list-uploader
- admin/options: less spam in your log thanks to the new "log api requests" and "log interface loading" options
- several fixes

beta10 [hfs-windows.zip](https://github.com/rejetto/hfs/files/11917076/hfs-windows.zip)
beta9 [hfs-windows.zip](https://github.com/rejetto/hfs/files/11909444/hfs-windows.zip)
beta8 [hfs-windows.zip](https://github.com/rejetto/hfs/files/11840713/hfs-windows.zip)
alpha7 [hfs-windows.zip](https://github.com/rejetto/hfs/files/11826686/hfs-windows.zip)
alpha6 [hfs-windows.zip](https://github.com/rejetto/hfs/files/11782536/hfs-windows.zip)
alpha5 [hfs-windows.zip](https://github.com/rejetto/hfs/files/11656091/hfs-windows.zip)
alpha1 [hfs-windows.zip](https://github.com/rejetto/hfs/files/11562759/hfs-windows.zip)

**Comment by rejetto** [comment]
alpha5 [hfs-windows.zip](https://github.com/rejetto/hfs/files/11656091/hfs-windows.zip)

go to admin/plugins/search and install "thumbnails"

**Reply by deamon2k** [reply]
counter is broken

**Reply by rejetto** [reply]
thanks for reporting, i will work on it

**Reply by BussyBakks** [reply]
where is website of online plugins? i want manually install plugin!

**Reply by rejetto** [reply]
this is where you find plugins
https://github.com/search?q=topic%3Ahfs-plugin&type=repositories

to manually install plugins, you download and copy only the "dist" folder into hfs "plugins" folder. Rename "dist" to the name of the plugin.

**Reply by rejetto** [reply]
i've created a new page on the wiki https://github.com/rejetto/hfs/wiki/Plugins

**Comment by xrivo1** [comment]
il plugin " anteprime " va in conflitto con l'antidos in quanto genera parecchie richieste...........

**Reply by rejetto** [reply]
thanks for reporting

**Reply by rejetto** [reply]
what's your settings for antidos?

**Comment by BussyBakks** [comment]
error: config.yaml not found
![image](https://github.com/rejetto/hfs/assets/108082159/08a4c675-d5aa-497f-b7ed-d0288517a776)

**Reply by rejetto** [reply]
thanks for reporting.
Ignore that error, it's harmless.
I will fix it in next release.

**Comment by rejetto** [comment]
alpha6 [hfs-windows.zip](https://github.com/rejetto/hfs/files/11782536/hfs-windows.zip)

- new icons for image, video and audio files
- file menu: show #270

**Comment by rejetto** [comment]
alpha7 [hfs-windows.zip](https://github.com/rejetto/hfs/files/11826686/hfs-windows.zip)
- fix: zip #274 
- download progress #273 
- rename #251 
- new icons for image/video/audio

**Comment by rejetto** [comment]
beta8 [hfs-windows.zip](https://github.com/rejetto/hfs/files/11840713/hfs-windows.zip)
- file menu: uploader #271

**Reply by BussyBakks** [reply]
cant translate "tiles mode", it doesnt in file and "hfs-langs-en.json"
how to translate?
![image](https://github.com/rejetto/hfs/assets/108082159/548b4a10-f1c1-4cde-91a7-d84928812bcb)

**Reply by rejetto** [reply]
thanks for reporting. I forgot to add those strings, now I just did.
To say it all, you could translate anyway. The file en.json is just a reference, it's not really used. I could delete it and everything would still work.

**Comment by mitobin** [comment]
File display feature when upgrading to version 0.46.0 beta 8, the .webm format only outputs audio, not video images. at version 0.46.0 alpha6 working properly. hope you fix it

**Reply by rejetto** [reply]
I made a test with a webm file that I have, and it works.
Instead of show try open: if you don't see the video then your browser doesn't support the file, and that's it.

**Reply by mitobin** [reply]
I reinstalled and it works fine. Thanks for taking the time to assist

**Reply by rejetto** [reply]
i noticed that on a slow connection, the webm file i use for tests appears like it has no video, until it is fully retrieved from the server.
So maybe what you experienced was just slowness.

**Comment by rejetto** [comment]
beta9 [hfs-windows.zip](https://github.com/rejetto/hfs/files/11909444/hfs-windows.zip)

- safer: prevent session cookie to be used by other ip
- safer: no csrf in log
- langs/vi
- langs/ru updated
- langs/fr updated
- plugin/download-counter: ignore option
- plugin/list-uploader
- admin/options: less spam in your log thanks to the new "log api requests" and "log interface loading" options
- several fixes

**Reply by GeorgeGeorgeGe** [reply]
It works perfect.
Thank you very very much.

**Reply by BussyBakks** [reply]
yey

**Reply by mitobin** [reply]
I have selected Vietnamese language but it is not applied. How to use it?

**Reply by BussyBakks** [reply]
Force language to "vi", or owner dont change txt file to json file, maybe

**Reply by rejetto** [reply]
force language is not working, it's a bug, i will fix it

**Comment by rejetto** [comment]
beta10 [hfs-windows.zip](https://github.com/rejetto/hfs/files/11917076/hfs-windows.zip)
- several fixes
- uploader information stored also for folders

hopefully last beta for 0.46

**Reply by GeorgeGeorgeGe** [reply]
it works PERFECT!
Thanks a lot.

**Comment by rejetto** [comment]
finally released
https://github.com/rejetto/hfs/releases/tag/v0.46.0
